tutormfe/plugin.py

```diff
@@ -44,6 +44,12 @@
                 },
             },
         },
+        "DISCUSSIONS_MFE_APP": {
+            "name": "discussions",
+            "repository": "https://github.com/open-craft/frontend-app-discussions",
+            "port": 2002,
+            "version": "opencraft-release/nutmeg.2"
```

Comment:
This is pointing to the current commit in upstream `master`. If needed, we can use the app as it was in the nutmeg release, but this version works just as well.